styleguide/components/21-atoms/icons/icon/README.md

```diff
@@ -1,4 +1,11 @@
 # Gent icon font
 
+## General
+
 For more information on the use of the Gent icon font see the notes in the
 `Base -> Fonts section`
+
+## a11y
```

Comment:
Change to Web accessibility.
Not all readers are familiar with 'a11y'.

Reply:
Done.